Source/Initialization/WarpXInitData.cpp

```diff
@@ -636,13 +636,9 @@ WarpX::computeMaxStepBoostAccelerator() {
     const Real interaction_time_boost = (len_plasma_boost-zmin_domain_boost_step_0)/
         (moving_window_v-v_plasma_boost);
     // Divide by dt, and update value of max_step.
-    int computed_max_step;
-    if (do_subcycling){
-        computed_max_step = static_cast<int>(interaction_time_boost/dt[0]);
-    } else {
-        computed_max_step =
-            static_cast<int>(interaction_time_boost/dt[maxLevel()]);
-    }
+    const int computed_max_step = (do_subcycling)?
```

Comment:
```suggestion
    const auto computed_max_step = (do_subcycling)?
```
I don't remember which clang-tidy check. But one of them will want this to be `auto`.

Reply:
Possibly the `modernize-use-auto` . Thanks for the suggestion!